server/src/services/form.ts

```diff
@@ -142,17 +164,51 @@ export class DatabaseFormService implements IFormService {
 
   public async addResponse(
     ctx: Context,
+    respondent: string,
     formID: string,
     answers: string[]
-  ): Promise<void> {
-    return; // TODO
+  ): Promise<IResponse> {
+    const response = await Response.create({ respondent: respondent, answers: answers });
```

Comment:
Answers likely have to be constructed independently as they are a different type of object, and require a reference to the question

Reply:
You can just keep this method out of the PR, since it's sort of a different issue